.github/workflows/finish-hackpod.yml

```diff
@@ -10,13 +10,13 @@ jobs:
     runs-on: ubuntu-latest
 
```

Comment:
Can you remove all these extra files/directories 🙂 :
- .github/workflows
- roiScript

Everything else should be good then!